foe.py

```diff
@@ -0,0 +1,47 @@
+import tcod
+from settings import Settings
+
+
+class FallingForEternity:
+    """Main class for assets and behavior"""
+
+    def __init__(self):
+        """initialize the game create resources"""
+
+        self.settings = Settings()
+        # Load the font, a 32 by 8 tile font with libtcod's old character layout.
+        self.tileset = tcod.tileset.load_tilesheet(
+            "resources\\images\\dejavu10x10_gs_tc.png", 32, 8, tcod.tileset.CHARMAP_TCOD,
```

Comment:
Replace this with os.path so that it is compatible on both Windows and Unix systems, as we have developers using both.

Reply:
Sounds good.